test/src/main/java/org/apache/accumulo/test/shell/ShellIT.java

```diff
@@ -566,6 +568,18 @@ public void propFileNotFoundTest() throws IOException {
         "FileNotFoundException: " + fileName + "(No such file or directory)");
   }
 
+  @Test
```

Comment:
You could be using 

```
  @TempDir
  private static File tempDir;
```

See ServiceLockIT (that's one example I found where its used)